scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -809,12 +809,11 @@ task GatherVariantCallingMetrics {
     disks: "local-disk ${disk_size_gb} HDD"
     bootDiskSizeGb: 15
     preemptible: 1
-    noAddress: true
```

Comment:
is this change necessary?

Reply:
I thought this was reverted..

@yonghaoy

Reply:
ah, I throught i removed it. trying again